agent/version/version.go

```diff
@@ -22,10 +22,10 @@ package version
 // repository. Only the 'Version' const should change in checked-in source code
 
 // Version is the version of the Agent
-const Version = "1.66.2"
+const Version = "1.67.0"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "6e8c9a51"
+const GitShortHash = "d49ed882"
```

Comment:
should this be the last commit on dev? `https://github.com/aws/amazon-ecs-agent/commits/dev` i.e. `d1fa74b5`